docs/content/reference/migration/migration-0-22.md

```diff
@@ -38,3 +38,19 @@ Instead, there's now methods with the same name, i.e. `ViewCoordinates::RUB()`.
 As part of the switch to "eager archetype serialization" (serialization of archetype components now occurs at time of archetype instantiation rather than logging), we can no longer offer exposing the `Tensor` **archetype** as `ndarray::ArrayView` directly.
 
 However, it is still possible to do so with the `TensorData` component.
+
+### C++ `RecordingStream::send_column` no longer takes raw component collections
+
+Previously, `RecordingStream::send_column` accepted raw component collections.
+
+This is no longer the case and only `rerun::ComponentColumn` and anything else from from which
+a `Collection<ComponentColumn>` can be constructed is accepted.
+The preferred way to create `rerun::ComponentColumn`s is to use the new `columns` method on archetypes.
+
+For instance in order to send a column of scalars, you can now do this.
+```cpp
+rec.send_columns("scalars", time_column,
+    rerun::Scalar().with_many_scalar(scalar_data).columns()
+);
+```
+All [example snippets](https://rerun.io/docs/snippets/all) have been updated accordingly.
```

Comment:
What's this link? Was that meant to be a link to the snippet index perhaps?

Reply:
urgh how did I butcher this. yes!